reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+    - handle
+    - title
+    - price
+    properties:
+      handle:
+        type: string
+        description: handle of the shipping rate.
+      title:
+        type: string
+        description: The shipping method name.
+      price:
+        type: string
+        description: The price of the shipping rate.
+
+
+  shipping_line:
+    type: object
+    required:
+    - title
+    - price
+    - tax_lines
+    discriminator: custom
+    properties:
+      custom:
+        description: Indicates whether or not the shipping_line is a custom_shipping_line.  When set to `true` indicates that it is a custom_shipping_line.  
+        type: boolean
+        enum: 
+        - true
+        - false
+  
+      title:
+        type: string
+        description: The title of the shipping method.
+        readOnly: false
+      price:
+        type: integer
+        format: int64
+        description: The price of this shipping method.
+        readOnly: false
+      phone:
+        type: string
+        description: The phone number, if required as specified in the `shipping_rate`.
+      tax_lines:
+        type: array
+        readOnly: false
+        items:
+          $ref: '#/definitions/tax_line'
+      
+
+  regular_shipping_line:
+    allOf:
+    - $ref: '#/definitions/shipping_line'
+    - type: object
+    required:
+    - title
+    - price
+    - tax_lines
+    properties:
+      handle:
+        type: string
+        description: The handle of the shipping rate which was selected and applied
+    
+      
+  custom_shipping_line:
+    allOf:
+    - $ref: '#/definitions/shipping_line'
+    - type: object
+    required:
+    - title
+    - price
+    - tax_lines
+    properties:
+      handle:
+        type: string
+        description: Nil for custom lines.
+         
+
+  metafield:
+    type: object
+    properties:
+      id:
+        type: integer
+        format: int64
+        description: Unique numeric identifier for the metafield.
+
+  tax_line:
+    type: object
+    required:
+    - title
+    - rate
+    - price
+    properties:
+      title:
+        type: string
+        description: The name of the tax.
+      rate:
+        type: string
+        description: The rate of tax to be applied.
+      price:
+        type: string
+        description: The amount of tax to be charged.
+
+  discount:
+    type: object
+    required:
+    - description
+    - value
+    properties:
+      description:
+        type: string
+        description: Reason for the discount.
+      value:
+        type: string
+        description: The value of the discount. If the type of the discount is `fixed_amount`, then this is a fixed dollar amount, if the type is `percentage`, then this is the percentage. 
+      type:
+        type: string
+        description: The type of the discount. Can be one of - `percentage` or `fixed_amount`. `Fixed_amount` is default value.
+        enum:
+        - percentage
+        - fixed_amount (default)
```

Comment:
Just checked and we made a mistake and didn't update our design doc. There's no defaulting behaviour. This is a required field